getgauge/util.py

```diff
@@ -2,7 +2,7 @@
 
 PROJECT_ROOT_ENV = 'GAUGE_PROJECT_ROOT'
 STEP_IMPL_DIR_ENV = 'STEP_IMPL_DIR'
-STEP_IMPL_DIR_NAME = os.getenv(STEP_IMPL_DIR_ENV) or 'step_impl'
+STEP_IMPL_DIR_NAMES = os.getenv(STEP_IMPL_DIR_ENV).split(',') if os.getenv(STEP_IMPL_DIR_ENV) else ['step_impl']
```

Comment:
trim dir names after splitting. consider scenario `STEP_IMPL_DIR=src, test, custom`.
Might be a good idea to have it as a function instead of constant since there is a bit of logic envolved.